uber/templates/registration/form.html

```diff
@@ -146,6 +146,27 @@
         </div>
     </div>
 
+    <div class="form-group">
+        <label for="admin_status" class="col-sm-2 control-label">Admin</label>
+        <div class="col-sm-6">
+            {% if attendee.admin_account %}
+                <p name="admin_status">True</p>
+            {% else %}
+                <p name="admin_status">False</p>
+            {% endif %}
+        </div>
+    </div>
+
+    {% if attendee.admin_account %}
+        <div class="form-group">
+            <label for="admin_options" class="col-sm-2 control-label">Permissions</label>
+            <div class="col-sm-6">
+                {% checkgroup attendee.admin_account.access %}
```

Comment:
Depends on what you want to display.  ``attendee.admin_account.access_labels`` should return a list, so you could theoretically do something like

```
{{ attendee.admin_account.access_labels|join:' / ' }}
```